src/sentry/static/sentry/app/views/settings/incidentRules/metricField.tsx

```diff
@@ -41,6 +42,12 @@ const getFieldOptionConfig = (dataset: Dataset) => {
         return ['tags[sentry:user]', 'string'];
       }
 
+      if (organization.features.includes('measurements')) {
```

Comment:
Not sure how alerts handles this but I'd lean towards splitting measurements out from fields as they may require special handling as we see here. So maybe another list for just `measurements` in the config.

Reply:
As an aside, `generateFieldOptions` accepts a list of `measurementKeys` along side `fields`, so you can make use of that.

Reply:
Should be addressed in https://github.com/getsentry/sentry/pull/21498/commits/524d080acc0dbde1646fcd66d091a5ab49412ff0